sites/fast-website/src/public/style.css

```diff
@@ -36,21 +40,6 @@ body {
   background-color: #3A3A3A;

```

Comment:
This should be using color recipes now that John has the design system provider up

Reply:
I will just delete it. It a temporary background to show a sectioned space.